glfs.c

```diff
@@ -772,10 +776,16 @@ static int tcmu_glfs_reconfig(struct tcmu_device *dev,
 			/* Let the targetcli command return success */
 			ret = 0;
 		} else if (st.st_size != cfg->data.dev_size) {
-			tcmu_dev_err(dev,
-				     "device size and backing size disagree: device %"PRId64" backing %lld\n",
-				     cfg->data.dev_size, (long long) st.st_size);
-			ret = -EINVAL;
+			/*
+			 * Currently we cannot update the size to kernel during
+			 * initialization, because it will be overrided by kernel
+			 * after it get the genl reply. Here will just allow it
+			 * to load the device and we will run gluster-block tool
+			 * to update the size.
+			 */
+			tcmu_dev_info(dev,
+				      "device size and backing size disagree:device %lld backing %lld\n",
+				      (long long)cfg->data.dev_size, (long long) st.st_size);
```

Comment:
shouldn't this be warning? something like below would be nice:
```
tcmu_dev_warn(dev,
              "device size (%lld) and backing file size (%ldd) not matching\n"
              (long long)cfg->data.dev_size, (long long) st.st_size);
```

Reply:
Make sense and will fix it.